hail/src/test/scala/is/hail/SparkSuite.scala

```diff
@@ -19,13 +19,13 @@ object SparkSuite {
 }
 
 class SparkSuite extends TestNGSuite {
-  lazy val hc: HailContext = SparkSuite.hc
+  val hc: HailContext = SparkSuite.hc
```

Comment:
The reason for making these not lazy anymore is because the `TableAggregate` execute method calls `HailContext.get` and will error out in the test suite because `hc` hasn't been called. I checked with @cseed and he thought this change should be fine. We had it lazy originally because creating the context is slow. But we think we've fixed it since then. Maybe we should compare how long it takes the test suite to run after this change compared to before.

Reply:
isn't it still slow? We could also add something to ensure initialization to these that need it

Reply:
I would prefer that these remain lazy if possible.

Reply:
I don't know why this would be significantly slower.  Only one context is ever created the first time a SparkSuite test is run.